packages/common/core-interfaces/src/logger.ts

```diff
@@ -46,13 +46,18 @@ export interface ITelemetryBaseEvent extends ITelemetryProperties {
 }
 
 /**
- * Enum to specify a level to the log to filter out logs based on the level.
+ * Specify levels of the logs.
  */
-export const enum LogLevel {
-	verbose = 10, // To log any verbose event for example when you are debugging something.
-	default = 20, // Default log level
-	error = 30, // To log errors.
-}
+export const LogLevels = {
+	verbose: 10, // To log any verbose event for example when you are debugging something.
+	default: 20, // Default log level
+	error: 30, // To log errors.
+} as const;
+
+/**
+ * Specify a level to the log to filter out logs based on the level.
+ */
+export type LogLevel = typeof LogLevels[keyof typeof LogLevels];
```

Comment:
```suggestion
export type LogLevels = typeof LogLevels[keyof typeof LogLevels];
```

Reply:
I renamed LogLevel to LogLevelType and LogLevels is the const object.

Reply:
Ok, cool:) I was wondering if we had to use different names.